openlane/steps/openroad.py

```diff
@@ -1806,10 +1806,15 @@ class CTS(ResizerStep):
             ),
             Variable(
                 "CTS_MAX_CAP",
-                Decimal,
-                "Defines the maximum capacitance, used in CTS.",
+                Optional[Decimal],
+                "Overrides the maximum capacitance used in CTS.",
                 units="pF",
-                pdk=True,
```

Comment:
why isn't it a PDK variable?

Reply:
It's a user override- CTS can get the values from the liberty file, so no need to declare it as a PDK config